CMakeLists.txt

```diff
@@ -2,13 +2,20 @@
 # CMake Project for REGRADER Plugin #
 #####################################
 
+# uncomment to build as VST2.4 instead of VST3.0 (provides wider DAW compatibility)
+#set(SMTG_CREATE_VST2_VERSION "Use VST2" ON)
+
+# uncomment to build Audio Unit (requires macOS and Xcode)
+#option(JAMBA_ENABLE_AUDIO_UNIT "Enable Audio Unit" ON)
+
+#######################
+# DO NOT CHANGE BELOW #
+#######################
+
 project(REGRADER)
-cmake_minimum_required(VERSION 3.0.0)
+cmake_minimum_required(VERSION 3.12)
 set(CMAKE_COLOR_MAKEFILE ON)
-set(CMAKE_CXX_STANDARD 11)
-
-# path to Steinberg VST3 SDK
-set(VSTSDK_PATH "/Library/VST_SDK/VST3_SDK" CACHE PATH "Path to VST3 SDK")
+set(CMAKE_CXX_STANDARD 14)
 
```

Comment:
Should now be provided via command line argument to prevent editing of checked in files.